cmd/action/internal/build/build.go

```diff
@@ -0,0 +1,5 @@
+package build
+
+// Version is the atlas CLI build version
+// Should be set by build script "-X 'ariga.io/atlas/cmd/action/internal/build.Version=${version}'"
+var Version string
```

Comment:
Why do we need a separate package for this variable? i.e. why not hold it in the version.go file?